HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectEntity.cs

```diff
@@ -21,16 +23,19 @@ public MapObjectEntity(MapObjectTypes mapObjectType, string description)
             Description = description;
         }
 
-         public MapObjectEntity(string description)
+        public SolidColorBrush getColor()
```

Comment:
Having get and set methods isn't really in accordance with C# conventions, could these be turned into properties?

Reply:
### Proposal

You have this:

![image](https://user-images.githubusercontent.com/45834270/98434445-f07d6180-20cf-11eb-87ba-16c7a5ee3a00.png)

You could do this:

![image](https://user-images.githubusercontent.com/45834270/98434316-dabb6c80-20ce-11eb-9013-d7ca57b51fa0.png)

or maybe this (if you do not have 'setter' )

![image](https://user-images.githubusercontent.com/45834270/98434327-e60e9800-20ce-11eb-997c-04150997c1a6.png)

### Next proposal

You have this:

![image](https://user-images.githubusercontent.com/45834270/98434362-1eae7180-20cf-11eb-835b-866ecd2e0049.png)

you could do this:

![image](https://user-images.githubusercontent.com/45834270/98434429-adbb8980-20cf-11eb-9183-ff6f2bd2166b.png)


You also in previous code don't have a color for 'WC' so you should add one.

Reply:
@vlaksi  I have already added colour for WC in my PR for g2 branch that waits for approval, so that shouldn't be a problem.

Reply:
Great @VasilijeBursac , then we only need to move method getColor to ObjectTypeColor property.